### PR TITLE
grid docs

### PR DESCRIPTION
**Pull request by matt3224**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -350,7 +348,216 @@
...
+                        <code class="language-sass">
+                            .mySuperGrid {
+                                @include _nu-object__grid();
+                            }
+                        </code>
+                    </pre>
+
+                    <div class="u-h6  u-mT-32  u-mB-16" id="configuringthemixin">Configuring the mixin</div>
+
+                    <table class="table">
```
Comment (darbymanning) [inline index.html]: Are we missing table element styles? Perhaps my build is wrong somehow?

screenshot 2019-02-28 at 10 03 16

**Comment by darbymanning** [comment]
Ah also, should we display the grid visually as we do on the testing page?

File: index.html (hunk 1 above)
Comment (matt3224) [reply index.html]: Yea we need some table styles, we spoke about it some time ago in regards to making them responsive

**Comment by matt3224** [comment]
Also yes we should illustrate the grid, i'll add that in. For dependencies, it may make more sense for the framework atleast to import them into the files like grid directly which would nullify the dependencies. This could work like importing a non-global polyfill in js and I can't see a downside to this, can you?

**Pull request merged**